packages/client/README.md

```diff
@@ -114,257 +203,284 @@ const msg = {
+...
+// assign a stream to a storage node
+await stream.addToStorageNode(STREAMR_STORAGE_NODE_GERMANY)
+// remove the stream from a storage node
+await stream.removeFromStorageNode(STREAMR_STORAGE_NODE_GERMANY)
+// fetch the storage nodes for a stream
+const storageNodes = stream.getStorageNodes()
+```
 
-| Name                                      | Description                                                                                                                                                                                                                                                                   |
-| :---------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| update()                                  | Updates the properties of this stream object by sending them to the API.                                                                                                                                                                                                      |
-| delete()                                  | Deletes this stream.                                                                                                                                                                                                                                                          |
-| getPermissions()                          |                                                                                                                                                                                                                               |
-| hasPermissions()                          | |
-| grantPermissions(              )          | |
-| revokePermissions()                       | |
-| detectFields()                            | Updates the stream field config (schema) to match the latest data point in the stream.                                                                                                                                                                                        |
-| publish(message, timestamp, partitionKey) | Publishes a new message to this stream.                                                                                                                                                                                                                                       |
 
-## Data Unions
+### Data Unions
 
 This library provides functions for working with Data Unions. Please see the [TypeScript generated function documentation](https://streamr-dev.github.io/streamr-client-javascript/classes/dataunion_dataunion.dataunion.html) for information on each Data Union endpoint.
```

Comment:
The link is not valid (and it is a API doc, not "TypeScript generated function documentation")